<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>160</width>
      <height>480</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="cubeButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>16</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Cube</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sphereButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>132</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>Sphere</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cylinderButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>248</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string>Cylinder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="coneButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>364</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Cone</string>
     </property>
    </widget>
   </widget>
   <widget class="QVTKOpenGLNativeWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
